mocha/utils/validators.js

```diff
@@ -23,3 +23,16 @@ exports.validateStatusCodeAndSuccess = function (
     .to.have.a.property("success")
     .to.be.equal(successMessage);
 };
+
+exports.validateStatusCodeAndOk = function (
```

Comment:
I think that we will have to add more generic validator that takes dict: key-value and validate it.